packages/ripple/README.md

```diff
@@ -24,6 +24,20 @@ with CSS:
 import '@material/react-ripple/dist/ripple.css';
 ```
 
+You'll also need to include these sass mixins on the element. Please also refer to [Advanced Sass Mixins](https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple/README.md#sass-apis) to customize further.
+
+```sass
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+
+// refer to element in Javascript portion below
+.ripple-icon-component {
```

Comment:
This code should be under "with Sass" because I think this doesn't apply at all to users only using the `dist/ripple.css`